omod/src/main/java/org/openmrs/module/owa/filter/OwaFilter.java

```diff
@@ -38,8 +41,16 @@ public void init(FilterConfig fc) throws ServletException {
 	@Override
 	public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain) throws IOException, ServletException {
 		HttpServletRequest request = (HttpServletRequest) req;
-		String owaBasePath = Context.getAdministrationService().getGlobalProperty(AppManager.KEY_APP_BASE_URL,
-		    DEFAULT_BASE_URL);
+		
+		String owaBasePath = "";
+		
+		try {
+			owaBasePath = Context.getAdministrationService()
+			        .getGlobalProperty(AppManager.KEY_APP_BASE_URL, DEFAULT_BASE_URL);
+		}
+		catch (Exception e) {

```

Comment:
Can we avoid making this a generic exception and instead use the specific class thrown?